### PR TITLE
Fixes #1731 - Remove all Browserstack related content from website and repo.

### LINKED ISSUE #1731: Remove Browserstack logo from website.
Now that we decided not to use it, better to remove the stuff from the website and anywhere that it's referenced.

### PR DESCRIPTION
**Pull request by brizental**
r? @miketaylr

### CODE REVIEW

File: webcompat/static/css/development/components/threesteps.css [hunk 1]
```diff
@@ -38,21 +38,6 @@
   transform-origin: center;
 }
 
-.wc-Browserstack {
-  width: 100%;
-  background-image: url("../../../img/browserstack_logo.svg");
```
Comment (miketaylr) [inline webcompat/static/css/development/components/threesteps.css]: Can we delete the svg file as well @ https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/img/browserstack_logo.svg?

Comment (brizental) [reply webcompat/static/css/development/components/threesteps.css]: uuh, forgot about that.

**Comment by brizental** [comment]
@miketaylr i think now it's alright.

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**